### ISSUE TITLE
Allow unbounded flamegraph sampling

### ISSUE THREAD

**Issue by masklinn** [benfred/py-spy#60]
When trying to sample desktop or web applications, the triggers are generally some user interaction until the end of whatever it did, with the "whatever" being only loosely specified time-wise possibly ranging from a fraction of a second to several minutes (e.g. a complex import process).

The current time-bound sampling duration is really inconvenient for this, it would be nice if like the "top" mode it were possible to start sampling and have the sampling only end when e.g. C-c is sent to py-spy.

**Comment by benfred** [comment]
Hrm - I think I messed up somewhere, this is supposed to be happening right now (with this commit https://github.com/benfred/py-spy/commit/290584dde76834599d66d74b64165dfe9a357ef5).

I just tested the released version on pip (0.1.8) and it looks like this commit wasn't included. I'll make sure this gets included in the next release, I'm honestly not sure why it's not there (best guess is that I packaged the release from my OSX laptop, and hadn't pulled that commit which originated on my linux desktop).
